### ISSUE TITLE
Add import function to load a previously created fsm

### ISSUE THREAD

**Issue by jessetilro** [evanw/fsm#8]
It is already possible to store and later load back in a diagram by manipulating the data stored in the browser's local storage. However, doing it through the interface would make it easier.

I would like to suggest adding an export function for JSON (practically dumping the local storage data to a file), in combination with an import function (simply loading this data from a file back into local storage).

**Comment by roman-ku** [comment]
+1 I would find it useful as well.

**Comment by gbroques** [comment]
There's a fork with this functionality at https://merfoo.github.io/fsm/

**Comment by jessetilro** [comment]
@gbroques Thanks, interesting! Too bad this functionality wasn't merged into the upstream repository. Just for further reference, the fork repository is located at https://github.com/Merfoo/fsm.

**Comment by gbroques** [comment]
Yup it's a shame the upstream repository is missing this functionality.

Merfoo's fork even lets you resize the canvas, and press <kbd>Ctrl-Z</kbd> / <kbd>Ctrl-Y</kbd> for Redo / Undo.
